data/net/minecraft/commands/arguments/blocks/BlockInput.mapping

```diff
@@ -7,5 +7,6 @@ CLASS net/minecraft/commands/arguments/blocks/BlockInput
 		ARG 1 level
 		ARG 2 pos
 		ARG 3 flags
+	METHOD test (Ljava/lang/Object;)Z
```

Comment:
Please revert this change and the others similar to it in the same package, as this adds nothing. (This will likely be removed by a `sanitizeData` pass.)